docs/docs/guides/smart_contracts/writing_contracts/notes/custom_note.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Using custom notes in Aztec.nr
+---
+
+It may be useful to write a custom note if you want to use a specific type of private data or struct that does not have a default implementation in Aztec.nr. Using a custom note means that the entire note is nullified with one nullifier, which may be better for your circumstances than having multiple notes that need nullified. 
+
+If you want to work with values or addresses, you can check out [ValueNote](./value_note.md) or [AddressNote](./address_note.md). 
+
+## Define a custom note
```

Comment:
As above, consider if "note type" is clearer.

Reply:
updated all "custom note" to "custom note type"